pkg/kubelet/dockershim/docker_sandbox.go

```diff
@@ -45,9 +45,18 @@ const (
 // Note: docker doesn't use LogDirectory (yet).
 func (ds *dockerService) RunPodSandbox(config *runtimeApi.PodSandboxConfig) (string, error) {
 	// Step 1: Pull the image for the sandbox.
-	// TODO: How should we handle pulling custom pod infra container image
-	// (with credentials)?
-	image := defaultSandboxImage
+	var (
+		image string
+	)
+	infraContainerImage := ds.podInfraContainerImage
+	if len(infraContainerImage) != 0 {
+		image = infraContainerImage
+	} else {
+		image = defaultSandboxImage
+	}
+
+	// NOTE(harryz) Assume user to handle custom pod infra container image pulling by following:
```

Comment:
Is this a TODO or just a regular comment? If it's the latter, there is no need to add your name here (as there is no action item). 

nit: Change it to `To use a custom sandbox image in a private repository, users need to configure the nodes with credentials properly. http://kubernetes.io/docs/user-guide/images/#configuring-nodes-to-authenticate-to-a-private-repository`